src/home_robot/home_robot/utils/data_tools/loader.py

```diff
@@ -125,14 +125,18 @@ def process_files(self, files):
                     if not self.trial_list or (
                         self.trial_list and key in self.trial_list
                     ):
-                        # Open the trial and extract metadata
-                        trial = self.Trial(key, filename, self, h5_trial)
-                        # get trial config, look for "demo_status" key and check if it is s/f
-                        if self.verbose:
-                            print("trial =", key, trial.length)
-                        lens.append(trial.length)
-                        # Bookkeeping for all the trials
-                        self.trials.append(trial)
+                        if (
+                            "demo_status" in h5_trial.keys()
```

Comment:
What is demo status

Reply:
Whether the episode was successful or not. Demonstrator feeds this in during data collection.

Reply:
add a comment to the code then, or rename the field to make it clear what's happening

Reply:
This still does not explain what demo status is

Reply:
Ah thanks for the call out. I explained this in H5 scripts but didn't realize I missed this comment!